addons/pos_self_order/static/src/app/pages/combo_page/combo_page.js

```diff
@@ -60,6 +61,17 @@ export class ComboPage extends Component {
         );
     }
 
+    isEveryValueSelected() {
+        return Object.values(this.state.selectedValues).every((value) => !value);
```

Comment:
isn't this what we need in fact ?

```suggestion
        return Object.values(this.state.selectedValues).every((value) => value);
```
Then we would have 
` t-att-disabled="!this.isEveryValueSelected()  or ...`

Reply:
The point is just that the name of the function should match it's contents

Reply:
i know that we already have the weird code on `product_page.xml`, but that's something that we should eventually change

Reply:
Done @vlst-odoo , let me know what you think.